Homework 3/Homework 3/Program.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System.Text;
+
+namespace Homework_3
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            bool TrueFalse = false;
+            User user1 = new User();
+            int age = 0;
+            while (true)
+            {
+                Console.Write("Enter your age: ");
+                try
+                {
+                    age = Convert.ToInt32(Console.ReadLine());
+                }
+                catch (Exception ex)
+                {
+                    Console.WriteLine(ex.Message);
+                }
+                if (age >= 0)
+                {
+                    user1 = new User(age);
+                    Console.Clear();
+                    break;
+                }
+                else
+                {
+                    Console.WriteLine("You`ve entered it incorrect, try again");
+                }
+            }
+            string firstName = "", lastName = "";
+            while (true)
```

Comment:
You can allocate this piece of code (while statement) in a separate method to avoid duplicates. So it could be like this:
private static string GetUserInput ( ) { }
and return a user choice from it (string firstName = GetUserInput())
something like that.